dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -34,45 +62,59 @@ std::string get_serialized_log(int signal,
                                const char *backtrace) {
     std::string serializedLog;
     serializedLog.append("{ ");
-    serializedLog.append(R"("message": ")").append(GENERIC_MESSAGE).append("\",");
-    serializedLog.append(R"("service": ")").append(SERVICE_NAME).append("\",");
-    serializedLog.append(R"("logger.name": ")").append(LOGGER_NAME).append("\",");
-    serializedLog.append(R"("env": ")").append(ENVIRONMENT).append("\",");
-    serializedLog.append(R"("status": ")").append(EMERGENCY_STATUS).append("\",");
+    if (!rum_context.application_id.empty()) {
+        serializedLog.append(R"("application_id": ")").append(rum_context.application_id)
+                .append("\",");
+    }
+    if (!rum_context.session_id.empty()) {
+        serializedLog.append(R"("session_id": ")").append(rum_context.session_id)
+                .append("\",");
+    }
+    if (!rum_context.view_id.empty()) {
+        serializedLog.append(R"("view.id": ")").append(rum_context.view_id)
+                .append("\",");
+    }
+    // these values are either constants or they are marked as NonNull in JVM so we do not have
+    // to check them here.
+    char tags[55]; // max 50 characters for the environment name
```

Comment:
Yes I know, I found quite reasonable (50 characters) in any case if this is longer it will not crash it will just strip the string . Maybe I should make this 100 just to be sure.